testing/environments/cloud/variables.tf

```diff
@@ -1,13 +1,13 @@
 ## Deployment configuration
 
 variable "ess_region" {
-  default     = "gcp-us-west2"
-  description = "Optional ESS region where the deployment will be created. Defaults to gcp-us-west2"
+  default     = "gcp-us-west1"
```

Comment:
We cannot use `gcp-us-west1` region because we don't publish snapshots there. Any reason why it was switched from `gcp-us-west2`?

Reply:
Evidently, `gcp-us-west2` is not a supported region: https://www.elastic.co/guide/en/cloud/current/ec-regions-templates-instances.html.  I tried it anyway initially and was getting the same error as before from the ESS API (via `terraform apply`):

```
│ Error: failed creating deployment: 9 errors occurred:
│ 	* api error: deployments.elasticsearch_using_legacy_dt: The referenced deployment template in elasticsearch plan [gcp-compute-optimized] is no longer available.
│ 	* api error: deployments.topology_element_using_legacy_ic: The referenced instance configuration [gcp.coordinating.1] in topology element [coordinating] is no longer available .
│ 	* api error: deployments.topology_element_using_legacy_ic: The referenced instance configuration [gcp.data.highcpu.1] in topology element [hot_content] is no longer available .
│ 	* api error: deployments.topology_element_using_legacy_ic: The referenced instance configuration [gcp.data.highstorage.1] in topology element [cold] is no longer available .
│ 	* api error: deployments.topology_element_using_legacy_ic: The referenced instance configuration [gcp.data.highstorage.1] in topology element [warm] is no longer available .
│ 	* api error: deployments.topology_element_using_legacy_ic: The referenced instance configuration [gcp.es.datafrozen.n1.64x10x95] in topology element [frozen] is no longer available .
│ 	* api error: deployments.topology_element_using_legacy_ic: The referenced instance configuration [gcp.master.1] in topology element [master] is no longer available .
│ 	* api error: deployments.topology_element_using_legacy_ic: The referenced instance configuration [gcp.ml.1] in topology element [ml] is no longer available .
```

So I had to switch it to `gcp-us-west1`.

Reply:
@ycombinator did it change recently, I used this Terraform script with `gcp-us-west2` a couple of months ago and it worked just fine.

Using any region that does not have snapshots (any other than `gcp-us-west2`) is not helpful because we always test the latest snapshot.

Reply:
> did it change recently?

@gersonsosa might be able to answer this question.